nemo_rl/models/generation/sglang/sglang_worker.py

```
@@ -0,0 +1,734 @@
+        
+        try:
+            if hasattr(self, "session") and self.session is not None:
+                try:
+                    async def close_session():
+                        await self.session.close()
+                        if self.connector is not None:
+                            await self.connector.close()
+                    
+                    self.async_loop_thread.run(close_session())
+                    print(f"[SGLang Worker] Rank {self.global_rank} aiohttp session closed.")
+                except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)